Bluepill-cli/Bluepill-cli/Bluepill/BPExecutionContext.m

```diff
@@ -8,7 +8,41 @@
 //  WITHOUT WARRANTIES OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License.
 
 #import "BPExecutionContext.h"
+#import "BPUtils.h"
+
+@interface BPExecutionContext ()
+
+@property (nonatomic, strong) BPWriter *simulatorLogWriter;
+
+@end
 
 @implementation BPExecutionContext
 
+- (BPWriter *)simulatorLogWriter {
```

Comment:
Just to be more consistent, just make writer public in header and init the writer in `Bluepill.m` and init in `createContext`